Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,128 @@
ernal ODBC database","34120","Failure","Drop","Terminated","End","ERROR","Low","Profiler failed to get the connection to NAC Manager","Profiler sends a notification event to NAC Manager, but the notification fails because could not connect to NAC Manager","34147","Failure","Drop","Terminated","End","WARN","Low","JGroups TLS Handshake Failed","JGroups TLS Handshake Failed","34148","Success","Allow","","Start","INFO","Informational","JGroups TLS Handshake Succeeded","JGroups TLS Handshake Succeeded","34149","Failure","Drop","Terminated","End","WARN","Low","HTTPS TLS Handshake Failed","HTTPS TLS Handshake Failed","34150","Success","Allow","","Start","INFO","Informational","HTTPS TLS Handshake Succeeded","HTTPS TLS Handshake Succeeded","34159","Success","Allow","","Start","INFO","Informational","LDAPS connection established successfully","LDAPS connection established successfully","34160","Success","Allow","","Start","INFO","Informational","LDAPS connection terminated successfully","LDAPS connection terminated successfully","34161","Failure","Drop","Terminated","End","WARN","Low","LDAPS connection establishment failed with SSL error","LDAPS connection establishment failed with SSL error","34162","Failure","Drop","Terminated","End","WARN","Low","LDAPS connection terminated with SSL error","LDAPS connection terminated with SSL error","34163","Failure","Drop","Terminated","End","WARN","Low","LDAPS connection establishment failed with non-SSL error","LDAPS connection establishment failed with non-SSL error","34164","Failure","Drop","Terminated","End","WARN","Low","LDAPS connection terminated with non-SSL error","LDAPS connection terminated with non-SSL error","90062","Failure","Drop","Terminated","End","ERROR","Low","Cannot connect to Domain Controller","Cannot connect to Domain Controller","90063","Success","Allow","","Start","INFO","Informational","Successfully establish connection to Domain Controller","Successfully establish connection to Domain Controller","90066","Failure","Drop","Terminated","End","ERROR","Low","Lost connection with Domain Controller","Lost connection with Domain Controller","90078","Success","Allow","","Start","INFO","Informational","Closed connection to Domain Controller","Closed connection to Domain Controller","91082","Failure","Drop","Terminated","End","WARN","Low","RADIUS DTLS: Connection to OCSP server failed","RADIUS DTLS: Connection attempt to OCSP server failed.","11317","Failure","Drop","Terminated","End","WARN","Low","TrustSec SSH connection failed","ISE failed to establish SSH connection to a network device. Verify network device SSH credentials in the Network Device page are similar to the credentials configured on the network device. Check network device enabled ssh connections from ISE (ip address)","5405","Failure","Drop","Terminated","End","NOTICE","Low","RADIUS Request dropped","RADIUS request dropped","5406","Failure","Drop","Terminated","End","NOTICE","Low","TACACS+ Request dropped","TACACS+ request dropped"];
```

Comment:
In my opinion, for 60234 the connection has disconnected. It is not a failure but a valid state. Please make the changes accordingly if this is true.

Reply:
Considering that EventOriginalType for this event is ERROR then EventResult is Failure. But if you want we can change it

Reply:
I think in that case, leave it the way it is.